docs.html

```diff
@@ -783,6 +783,275 @@ <h5 class="component-subheading">Form Example</h5>
+            <li>List item 3</li>
+        </ol>
+        <ol class="list-style-upper-alpha list-container">
+            <li>List item 1</li>
+            <li>List item 2</li>
+            <li>List item 3</li>
+        </ol>
+        <ol class="list-style-lower-alpha list-container">
+            <li>List item 1</li>
+            <li>List item 2</li>
+            <li>List item 3</li>
+        </ol>
+    </div>
+<pre>
+    <code class="html">
+        <ol>
+            <li>List item 1</li>
+            <li>List item 2</li>
+            <li>List item 3</li>
+        </ol>
+        <ol reversed>
+            <li>List item 1</li>
+            <li>List item 2</li>
+            <li>List item 3</li>
+        </ol>
+        <ol class="list-style-upper-roman">
+            <li>List item 1</li>
+            <li>List item 2</li>
+            <li>List item 3</li>
+        </ol>
+        <ol class="list-style-lower-roman">
+            <li>List item 1</li>
+            <li>List item 2</li>
+            <li>List item 3</li>
+        </ol>
+        <ol class="list-style-upper-alpha">
+            <li>List item 1</li>
+            <li>List item 2</li>
+            <li>List item 3</li>
+        </ol>
+        <ol class="list-style-lower-alpha">
+            <li>List item 1</li>
+            <li>List item 2</li>
+            <li>List item 3</li>
+        </ol>
+    </code>
+</pre>
+<hr>
+    <h5 class="component-subheading">List with no styling</h5>
+        <div class="component-container">
+            <ul class="list-non-bullet list-container">
+                <li>List item 1</li>
+                <li>List item 2</li>
+                <li>List item 3</li>
+            </ul>
+        </div>
+<pre>
+    <code class="html">
+        <ul class="list-non-bullet">
+            <li>List item 1</li>
+            <li>List item 2</li>
+            <li>List item 3</li>
+        </ul>
+    </code>
+</pre>
+<hr>
+
+    <h5 class="component-subheading">Stacked List</h5>
+    <p class="component-desc">This is an unordered list without bullets. We can add desired component inside the li element.</p>
+    <div class="component-container">
+    <ul class="list-non-bullet">
+        <li>
+            <div class="notification flex-r">
+                Stacked Notification
+                <i class="m-left-auto far fa-times-circle"></i>
+            </div>
+        </li>
+        <li>
+            <div class="notification flex-r">
+                Stacked Notification
+                <i class="m-left-auto far fa-times-circle"></i>
+            </div>
+        </li>
+        <li>
+            <div class="notification flex-r">
+                Stacked Notification
+                <i class="m-left-auto far fa-times-circle"></i>
+            </div>
+        </li>
+    </ul>
+    </div>
+    
+<pre>
+    <code class="html">
+        <ul class="list-non-bullet">
+            <li>
+                <div class="notification flex-r">
+                    Stacked Notification
```

Comment:
You can reduce this extra div by directly using <li> as the container and try to give the "notification flex-r' class to <li> tag.

Reply:
Thanks for reviewing.